health/micro-ui/web/micro-ui-internals/packages/modules/campaign-manager/src/pages/employee/CreateChecklist.js

```
@@ -36,20 +43,26 @@ const CreateChecklist = () => {
+  const [loading_new, setLoading_New] = useState(true);
   let locale = Digit?.SessionStorage.get("initData")?.selectedLanguage || "en_IN";
   const { mutateAsync } = Digit.Hooks.campaign.useCreateChecklist(tenantId);
   const history = useHistory();
-  let data_mdms=[]
-  let template_data=[]
+  const [serviceCode, setServiceCode] = useState(null);
+  const [def_data, setDef_Data] = useState(null);
+
+  // const { questionData1, dispatchQuestionData } = useContext(QuestionContext);
+
+  let data_mdms = []
```

Biome

[error] This let declares a variable that is only assigned once.

'data_mdms' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)